_studio/mfx_lib/encode_hw/h265/src/mfx_h265_encode_vaapi.cpp

```diff
@@ -417,6 +418,10 @@ mfxStatus SetRateControl(
 
     rate_param->initial_qp = par.m_pps.init_qp_minus26 + 26;
 
+    //  MBBRC control
+    // Control VA_RC_MB 0: default, 1: enable, 2: disable, other: reserved
+    rate_param->rc_flags.bits.mb_rate_control = mbbrc & 0xf;
```

Comment:
Can't you directly use value from ```par``` here?
i.e. 
```
rate_param->rc_flags.bits.mb_rate_control = IsOn(par.m_ext.CO2.MBBRC) ? 1 : IsOff(par.m_ext.CO2.MBBRC) ? 2 : 0;
```

Reply:
I have corrected.